jenkins/test_orchestrator/JenkinsFile

```diff
@@ -0,0 +1,62 @@
+pipeline {
+    agent any
+
+    parameters {
+        string(name: 's3_bucket'),
+        string(name: 'opensearch_version'),
+        string(name: 'build_id'),
+        string(name: 'architecture')
+    }
+
+    stages {
+        stage('Execute Tests') {
+            steps {
+                // To run all three test jobs in parallel, we use "parallel" step
+                // https://jenkins.io/doc/pipeline/examples/#jobs-in-parallel
+                parallel (
+                    IntegTests: {
```

Comment:
Does this spin up 3 agent nodes?

Reply:
no, this doesn't create any infra. It should be created separately via cdk.

Reply:
Per #399 we should not use parallel, and instead run these stages serially until we can invest in this improvement.  As much as I know it would be a big perf boost, its not worth the stability trade-off.

Reply:
If the parallel infrastructure is not available, this will automatically kick these stages off in serial order.